src/robusta/patch/patch.py

```diff
@@ -29,6 +30,17 @@ def create_monkey_patches():
     # We added caching to search for the plugins only once
     logging.info("Creating yaml monkey patch")
     YAML.official_plug_ins = official_plug_ins
+    # The patched method is due to a bug in containerd that allows for containerImages to have no names
+    # which causes the kubernetes python api to throw an exception
+    logging.info("Creating kubernetes ContainerImage monkey patch")
+    V1ContainerImage.names = V1ContainerImage.names.setter(names)
+
+
+def names(self, names):
+    if names:
```

Comment:
This doesn't compile on my machine.

Reply:
there was a tab that got erased in the line below by acident